master/src/main/scala/org/apache/celeborn/service/deploy/master/quota/QuotaManager.scala

```diff
@@ -57,7 +57,7 @@ class QuotaManager(celebornConf: CelebornConf, configService: ConfigService) ext
       userIdentifier: UserIdentifier,
       value: Long,
       quota: Quota): (Boolean, String) = {
-    val exceed = (quota.diskBytesWritten > 0 && value >= quota.diskBytesWritten)
+    val exceed = (quota.diskBytesWritten >= 0 && value >= quota.diskBytesWritten)
```

Comment:
It seems that if the value is negative, it indicates that the quota is unlimited, which feels a bit counterintuitive. For numbers less than or equal to 0, it seems better to directly determine them as having no quota.
cc @leixm @AngersZhuuuu